src/components/App/App.js

```diff
@@ -7,6 +7,7 @@ import Header from '../Header/Header';
 import Footer from '../Footer/Footer';
 import Profiles from '../Profiles/Profiles';
 import Passport from '../Passport/Passport';
+import ChartView from '../ChartView/ChartView';
```

Comment:
Glad to see that this isn't throwing merge conflict errors anymore. This file's folder and imports were changed to uppercase and it appears to solve this problem.